src/modules/powerrename/dll/dllmain.cpp

```diff
@@ -224,6 +224,9 @@ class PowerRenameModule : public PowertoyModuleIface
         return 0;
     }
 
+    virtual void register_system_menu_helper(PowertoySystemMenuIface* helper) override { }
+    virtual void signal_system_menu_action(const wchar_t* name) override { }
+

```

Comment:
Please add 
```
  virtual void register_system_menu_helper(PowertoySystemMenuIface* helper) override { }
  virtual void signal_system_menu_action(const wchar_t* name) override { }
```
To the `example_powertoy` project as well.
It's not currently built but it's in the solution (even if we already planned to remove it from the solution,  we don't want to break it while it's still present).